### DISCUSSION TITLE
Issues When Saving A Page

### DISCUSSION THREAD

**Discussion by oldmanumby** [requarks/wiki#5088] [Error / Bug Report] [answered]
So, I have a fresh install of a wiki using Cloudron. The install seems fine, but when I create a page, or when I make changes to a page I get a series of errors (see attachments). The system seems to save the pages correctly, but the errors are odd. 

The name of the page I'm testing is called "Home", which shouldn't be a problem. No other pages have been created thus far and I've only adjusted the most basic of settings. Nothing else.

Suggestions?

<img width="398" alt="Screen Shot 2022-03-12 at 3 20 30 PM" src="https://user-images.githubusercontent.com/22547499/158035649-af08ab02-2f3e-4f4d-af58-b9452823efb4.png">

<img width="547" alt="Screen Shot 2022-03-12 at 3 20 46 PM" src="https://user-images.githubusercontent.com/22547499/158035652-d0281cdb-28df-468c-9208-39c82b32f376.png">

<img width="512" alt="Screen Shot 2022-03-12 at 3 20 56 PM" src="https://user-images.githubusercontent.com/22547499/158035656-2f5e06e8-5229-48eb-b69e-8373f29ae652.png">

<img width="590" alt="Screen Shot 2022-03-12 at 3 21 12 PM" src="https://user-images.githubusercontent.com/22547499/158035658-1d8b0793-225d-41be-8744-d94fdf7926a5.png">

**Comment by NGPixel** [comment, marked as answer]
Are you using a search engine other than the default basic one? This error means the search engine isn't configured correctly.

**Comment by oldmanumby** [comment]
Ah yes, that was it. I set up Algolia; not sure what the deal is though. All the correct info is populated in the fields. But when I switch back to "Database - Basic" the error is gone. THX! ... gotta love those easy fixes!
